pulp_rpm/app/viewsets.py

```diff
@@ -395,6 +784,16 @@ class PackageGroupViewSet(ReadOnlyContentViewSet):
     queryset = PackageGroup.objects.all()
```

Comment:
some content viewsets still are missing the policy, also consider adding queryset scoping to it https://github.com/pulp/pulp_file/pull/636/commits/8a728da81d70c08b6a0f89dd0f4d4fd3769942cf

Reply:
I'd postpone scoping for follow up work on rbac as implementation from core can change.